src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1341,6 +1341,64 @@
         az acr token update -n MyToken -r MyRegistry --scope-map MyNewScopeMap
 """
 
+helps['acr artifact-streaming'] = """
+type: group
+short-summary: Manage artifact streaming in any repositories or images.
+"""
+
+helps['acr artifact-streaming operation'] = """
+type: group
+short-summary: Allows you to manage the streaming artifact creation.
```

Comment:
NIT: Allows you to manage the streaming artifact creation for Azure Container Registry.
comment: Looks like the other summaries include ACR (so to be consistent).